transformerlab/routers/data.py

```diff
@@ -59,28 +59,24 @@ async def dataset_info(dataset_id: str):
 
 
 @router.get("/preview", summary="Preview the contents of a dataset.")
-async def dataset_preview(dataset_id: str):
+async def dataset_preview(dataset_id: str, offset: int = 0, limit: int = 10):
     d = await db.get_dataset(dataset_id)
-
+    datasetLen = 0
```

Comment:
https://peps.python.org/pep-0008/

PEP8:

Function names should be lowercase, with words separated by underscores as necessary to improve readability.